cotainr/container.py

```diff
@@ -147,14 +147,27 @@ def add_metadata(self):
             f.seek(0)
             json.dump(metadata, f)
 
+    def _create_file(self, *, env_file):
+        """
+        Create the Path/file `env_file` in the Singularity container.
```

Comment:
The docstring seems a bit misleading now that the scope of the method has been updated. At the same time it suggests to two different things to me:

1. You can create any path/file in the container.
2. You can only create an "env file".

The current implementation only seems to be able to create empty files in already existing folders.

We should probably rename `env_file` to `file` or `path`, depending on the scope of the method. The docstring should be more clear.